src/main/java/com/liveperson/faas/csds/CsdsWebClient.java

```diff
@@ -33,15 +33,16 @@ public String getDomain(String service) throws CsdsRetrievalException {
                 .map(BaseURI::getBaseURI)
                 .orElseThrow(() -> new CsdsRetrievalException(String.format(
                         "Service with name %s could not be found " +
-                                "in domains", service)));
+                                "in domains",
+                        service)));
     }
 
     private List<BaseURI> getCachedDomains() throws CsdsRetrievalException {
         if (cachedDomainsInitialized() && !cachedDomainsExpired()) {
             return cachedDomains;
         } else {
             BaseURIs baseURIs;
-            String csdsRetrivalUrl = String.format("http://%s/api/account/%s/service/baseURI.json?version=1.0",
+            String csdsRetrivalUrl = String.format("https://%s/api/account/%s/service/baseURI.json?version=1.0",
```

Comment:
This was breaking java client when using Alpha CSDS domain "adminlogin-a.liveperson.net". Seems that with http does not work on the client.